optax/assignment/_hungarian_algorithm.py

```diff
@@ -21,38 +21,42 @@
 
 
 def hungarian_algorithm(cost_matrix):
-  r"""The Hungarian algorithm for the linear assignment problem.
```

Comment:
first line should not exceed 80 chars and contain just the summary (https://google.github.io/styleguide/pyguide.html#381-docstrings). You can add a longer line below with the appropriate link